src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -374,4 +411,36 @@ public Command pointTowardsTranslation(DoubleSupplier x, DoubleSupplier y) {
                         getPose().getRotation().getRadians(),
                         getVelocity().omegaRadiansPerSecond)));
   }
+
+  public Command runModuleSteerCharacterizationCmd() {
+    return Commands.sequence(
+        this.runOnce(() -> SignalLogger.start()),
+        moduleSteerRoutine.quasistatic(Direction.kForward),
+        this.stopCmd(),
```

Comment:
Is this required?

Reply:
Sorry, literal this

Reply:
i think its more clear to use 'this' when invoking commands on a subsystem since it's explicitly doing an action to the subsystem, but am open to discussion

Reply:
Making sure I know. In other languages, unnecessarily using this can create retain cycles.

Reply:
a quick google search says that javas gc algo takes care of those